input/data/2018/2018ApJ...856...95A/tev-000148-lc-3.ecsv

```diff
@@ -0,0 +1,23 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: time_err, unit: MJD, datatype: float64}
```

Comment:
@GernotMaier - Here you have `time_err` in the LC.  We don't support / allow this at the moment.
What does this mean?
If it's a measure of a time interval, can you please change to `time_min` and `time_max` columns?
If actually you had a problem with your clock and there's an error associated with the time, that is pretty exceptional and I'd suggest to just note it as a comment and remove the `time_err` here to save us some work supporting / documenting it in gamma-cat.

Reply:
Replace it my time_min / time_max